microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -1052,33 +1004,21 @@ See also:
 
 Status:
 * This checkbox is present in Microsoft Edge Canary 125.
-* This checkbox is present in Microsoft Edge Stable 123.
+* This checkbox is present in Microsoft Edge Stable 124.
 
 
 <!-- ====================================================================== -->
-## Enable context menu that allows to modify trees in the Flame Chart
+## Enable the legacy tracing model when inspecting Chrome traces using the performance panel
 <!-- no (?) link -->
 
-_new as of Microsoft Edge 121_
-
-Adds a right-click menu that allows modifying trees in the flame chart in the **Performance** tool.  With this experiment enabled, you can right-click events in the flame chart of the **Performance** tool, and modify the chart at that location:
-
-![Context menu to modify flame chart trees](./index-images/context-menu-modify-flame-chart-trees.png)
+_new as of Microsoft Edge 126_
 
-The right-click menu potentially has the following menuitems, depending on context:
-* **Merge function**
-* **Collapse function**
-* **Collapse repeating descendants**
-* **Undo collapse function**
-* **Undo collapse repeating descendants**
-* **Undo all actions**
-* **Load profile**
-* **Save profile**
+Whether to enable the legacy tracing model when inspecting Chrome traces by using the **Performance** tool.
+<!-- todo: improve desc -->
```

Comment:
Need desc for experiment "Enable the legacy tracing model when inspecting Chrome traces using the performance panel"
@captainbrosset

Reply:
This experiment is not meant for real users to toggle. As far as I understand, it's more of an internal implementation convenience to test a new implementation of the tracing model that the Performance tool uses. It will get removed after a short period of time (at least that's the plan).

So, I think we should say something like: "An internal setting that controls whether to enable the legacy tracing model that the **Performance** tool uses. Do not use."

Would this be ok?

Reply:
```
An internal setting that controls whether to enable the legacy tracing model 
that the **Performance** tool uses.  Do not use.
```